lib/index.coffee

```diff
@@ -39,6 +39,8 @@ getImagesToRemove = (tree, reclaimSpace) ->
 			result.push(leaf)
 			size += leaf.size
 		leaf.removed = true
+
+	@metrics?.gauge('gc_number_images_to_remove', result.length)
```

Comment:
Please be sure to lint these metrics with `promtool` to conform to good naming practices. You can build that from here: https://github.com/prometheus/prometheus/tree/master/cmd/promtool and then lint it like so:

```
curl -s http://localhost:9090/metrics | promtool check metrics
```

obviously changing the URL to whatever you require.

Reply:
:+1:

Reply:
@xginn8 has a good point. All counters should end with `_total` in their name. All histograms should have a suffix indicating the unit in plural form (in this case, histograms will have `_milliseconds`)

Reply:
Re: https://prometheus.io/docs/practices/naming/

Reply:
Done! Thanks :)